Assets/Scripts/Augment/AugmentImplementations/KnockbackProjectileModifier.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using TransformExtensions;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class KnockbackProjectileModifier : MonoBehaviour, ProjectileModifier
+{
+    [SerializeField]
+    private float pushPower = 50f;
+
+    [SerializeField]
+    private GameObject stuckObject;
+
+    [SerializeField]
+    private float knockbackRadius;
+
+    private PlayerManager source;
+
+    public void Attach(ProjectileController projectile)
+    {
+        projectile.OnColliderHit += KnockAwayTarget;
+    }
+
+    public void Detach(ProjectileController projectile)
+    {
+        projectile.OnColliderHit -= KnockAwayTarget;
+    }
+
+    public void KnockAwayTarget(Collider collider, ref ProjectileState state)
+    {
+        var stuck = Instantiate(stuckObject, state.position, state.rotation, null);
+        stuck.transform.ParentUnscaled(collider.transform);
+        stuck.gameObject.GetComponent<KnockbackEffect>().KnockAwayTargets(pushPower, knockbackRadius, stuck.transform.position);
```

Comment:
So we have two different modifiers for knockback now?

Reply:
First implementation was for a spherical explosive knockback. The second is just two directional. In the case someone wants to use the first one, for instance a rocket launcher, they could use this method.

Reply:
Removed the file and stored it locally. It was unnecessary after some discussion with Fredrik.

Reply:
Mkay